CHANGES.md

```diff
@@ -1,5 +1,8 @@
 ## Version 2.1.4 (in development)
-
+* When saving a dataset to disk, cate now checks first if the needed keys for geospatial information
```

Comment:
```suggestion
* When caching a dataset to disk, cate now checks first if the needed attributes for geospatial information
```

Reply:
Here, we only resort to the information from the metadata when we cannot retrieve the resolution from the data.